test/__init__.py

```diff
@@ -151,15 +151,16 @@ def notBrotli() -> Callable[[_TestFuncT], _TestFuncT]:
     )
 
 
-def onlyZstd() -> Callable[[_TestFuncT], _TestFuncT]:
+def hasZstd() -> Callable[[_TestFuncT], _TestFuncT]:
```

Comment:
Why change `only` to `has`? `only` was fine, and consistent with other fixtures like `onlySecureTransport`

Reply:
Changed back to `only`.